packages/iov-keycontrol/src/keyring-entries/ed25519.ts

```diff
@@ -120,11 +120,19 @@ export class Ed25519KeyringEntry implements KeyringEntry {
     this.labelProducer.update(label);
   }
 
-  public async createIdentity(): Promise<LocalIdentity> {
-    const seed = await Random.getBytes(32);
-    const keypair = await Ed25519.makeKeypair(seed);
+  public async createIdentity(keypair?: Ed25519Keypair): Promise<LocalIdentity> {
```

Comment:
Hmmm... I kinda like to check existence at compile-time, but I understand too hard.